setup.py

```diff
@@ -17,7 +17,7 @@ def run(self):
         super().run()
 
 setup(
-    name='nml',
+    name='nml-test',
```

Comment:
temporary, I assume?

Reply:
From the PR description
> * [ ] Remove test code

Yes :p Necessary for testing upload to test.pypi.org, as nml already exists (surprisingly enough)